### PR TITLE
Handle client networking errors correctly

### PR DESCRIPTION
**Pull request by chaudum**
## Summary of the changes / Why this is an improvement

Print only a sensible error message instead the full traceback.

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: croud/rest.py [hunk 1]
```diff
@@ -79,8 +80,17 @@ def send(
...
         ) as session:
-            resp = await session.fetch(method, endpoint, body, params)
-            return await self._decode_response(resp)
+            try:
+                resp = await session.fetch(method, endpoint, body, params)
+            except ClientConnectorError as e:
+                message = (
+                    f"Failed to perform command on {e.host}. "
+                    f"Original error was: '{e}' "
+                    f"Did you specify an existing combination of --region and --env?"
```
Comment (MarkusH) [inline croud/rest.py]: Maybe rephrase to "Does the environment exist in the region you specified?"

**Comment by chaudum** [comment]
Hm, the required CI status of Travis does not report back.
